sdks/python/tests/test_requirements.txt

```diff
@@ -1,7 +1,11 @@
+pandas
```

Comment:
These libraries are needed for unit tests only and they are already added to unit tests requirements